grpcurl.go

```diff
@@ -629,7 +630,16 @@ func BlockingDial(ctx context.Context, network, address string, creds credential
 			TransportCredentials: creds,
 			writeResult:          writeResult,
 		}
+	} else {
+		creds = insecure.NewCredentials()
+	}
+
+	var err error
+	creds, err = xdsCredentials.NewClientCredentials(xdsCredentials.ClientOptions{FallbackCreds: creds})
```

Comment:
It seems like this code path should only be triggered if the `address` includes the `xds://` scheme, no?

Reply:
Actually, this is what I mentioned in my PR description. This is a noop if the target _isn't_ `xds:///`, so we can avoid the potentially awkward check for the prefix. I can add it if you want to have it explicitly

Reply:
I updated this to only do it if it's an `xds:///` target